packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -149,18 +151,10 @@ class ZapierBaseCommand extends Command {
       this.log(colors.gray(emptyMessage));
     } else {
       // data comes out of the formatter ready to be printed (and it's always in the type to match the format) so we don't need to do anything special with it
-      console.log(formatter(rows, headers));
+      console.log(formatter(rows, headers, showHeaders, hasBorder, style));
     }
   }
 
-  /**
-   * Print text in a list style.
-   * @param {string[][]} items
-   */
-  logList(items) {
-    this.log(renderList(items, { spacer: '\n', maxWidth: stdtermwidth }));
```

Comment:
Its only use case was logging a table-like list in the `validate` command. Since we no longer have `renderList`, `logList` is replaced by `logTable`.  The table-like list is really a table after all.

Reply:
![](https://cdn.zappy.app/766c7a3eda427cf898921a4f68945147.png)

Reply:
I found another use case in `oclif/commands/versions`!